src/commands/report.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const aws = require('aws-sdk');
+const lighthouse = require('lighthouse');
+const chromeLauncher = require('chrome-launcher');
+const { v1: uuidv1 } = require('uuid');
+const config = require('../config');
+// const { categories, deviceForms } = require('../constants');
+
+const s3 = new aws.S3({
+	accessKeyId: config('AWS_ACCESS_KEY_ID'),
+	secretAccessKey: config('AWS_SECRET_ACCESS_KEY'),
+	region: config('REGION')
+});
+
+const uploadFile = file => {
+	const uniqueId = uuidv1();
+	const fileName = `report-${uniqueId}.html`;
+	const fileType = 'text/html';
+	const storageClass = 'STANDARD';
+	const acl = 'public-read'
+
+	const params = {
+		Bucket: config('S3_BUCKET'),
+		Key: fileName,
+		Body: file,
+		StorageClass: storageClass,
+		ContentType: fileType,
+		ACL: acl
+	};
+
+	s3.upload(params, function(err, data) {
+		if (err) {
+			console.log('Something went wrong. Try again later.');
+		}
+			console.log(`File uploaded successfully. ${data.Location}`);
+	});
+
+};
```

Comment:
Still have to handle this error in the future when the report file fails to upload in the S3 bucket.